test/unit/ngsiv2/MQTT_receive_ngsiv2_measures-test.js

```diff
@@ -464,18 +464,18 @@ describe('MQTT: NGSIv2 Measure reception ', function () {
                 .matchHeader('fiware-service', 'smartgondor')
                 .matchHeader('fiware-servicepath', '/gardens')
                 .post(
-                    '/v2/entities?options=upsert',
-                    utils.readExampleFile('./test/unit/ngsiv2/contextRequests/ngsiv2PayloadMeasure.json')
-                )
-                .reply(204);
-            contextBrokerMock
-                .matchHeader('fiware-service', 'smartgondor')
-                .matchHeader('fiware-servicepath', '/gardens')
-                .post(
-                    '/v2/entities?options=upsert',
-                    utils.readExampleFile('./test/unit/ngsiv2/contextRequests/ngsiv2PayloadMeasure2.json')
+                    '/v2/op/update',
+                    utils.readExampleFile('./test/unit/ngsiv2/contextRequests/ngsiv2PayloadMeasure3.json')
                 )
                 .reply(204);
+            // contextBrokerMock
+            //     .matchHeader('fiware-service', 'smartgondor')
+            //     .matchHeader('fiware-servicepath', '/gardens')
+            //     .post(
+            //         '/v2/entities?options=upsert',
+            //         utils.readExampleFile('./test/unit/ngsiv2/contextRequests/ngsiv2PayloadMeasure2.json')
+            //     )
+            //     .reply(204);
```

Comment:
Leftover to be removed?

Reply:
Removed in https://github.com/telefonicaid/iotagent-json/pull/857/commits/8e4a280526e8ef46e9f7fd30c23135dafa754ca0